main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -80,7 +80,7 @@ object Resolver {
   /**
     * Performs name resolution on the given program `root`.
     */
-def run(root: NamedAst.Root, oldRoot: ResolvedAst.Root, changeSet: ChangeSet)(implicit flix: Flix): Validation[ResolvedAst.Root, ResolutionError] = flix.phase("Resolver") {
+  def run(root: NamedAst.Root, oldRoot: ResolvedAst.Root, changeSet: ChangeSet)(implicit flix: Flix): Validation[ResolvedAst.Root, ResolutionError] = flix.phase("Resolver") {
```

Comment:
I've thought about this change so many times 😄

Reply:
I get it on almost every PR that touches Resolver

Reply:
heh